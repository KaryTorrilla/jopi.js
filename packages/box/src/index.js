import styled from 'styled-components'
import {
  space,
  color,
  layout,
  flexbox,
  border,
  typography,
  position,
  variant,
} from 'styled-system'
import css from '@styled-system/css'

const sx = props => css(props.sx)(props.theme)
const base = props => css(props.__css)(props.theme)
const theme = ({ tx = 'variants' }) =>
  variant({
    scale: tx,
  })

export const Box = styled.div(
  {
    boxSizing: 'border-box',
    margin: 0,
    minWidth: 0,
  },
  base,
  theme,
  sx,
  space,
  color,
  layout,
  flexbox,
  typography,
  border,
  position
)

export const Flex = styled(Box)({
  display: 'flex',
})
