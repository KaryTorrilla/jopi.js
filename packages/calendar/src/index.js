import React from 'react'
import FullCalendar from '@fullcalendar/react' // must go before plugins
import dayGridPlugin from '@fullcalendar/daygrid' // a plugin!
import '@oneloop/theme/styles/globals.css'
import '../styles/styles.css'

export const Calendar = () => {
  /*   const calendarOptions = {
      events: [],
      plugins: [timeGridPlugin],
      initialView: 'timeGridDay',
      nowIndicator: true,
      headerToolbar: {
        start: 'prev',
        center: 'title',
        end: 'next',
      },
      editable: true,
      eventResizableFromStart: true,
      dragScroll: true,
      dayMaxEvents: 2,
      navLinks: true,
    } */

  return (
    <div className='calendarContainer'>
      <FullCalendar
        plugins={[dayGridPlugin]}
        initialView="dayGridMonth"
      />
    </div>
  )
}
