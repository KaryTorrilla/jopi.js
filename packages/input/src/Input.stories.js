import React from 'react'

import Input from '.'

export default {
  component: Input,
  title: 'Design System|Input',
}

export const placeholder = () => <Input placeholder="Hello World!" />

export const error = () => (
  <Input placeholder="Hello World!" borderColor="danger" />
)

export const success = () => (
  <Input placeholder="Hello World!" borderColor="success" />
)

export const warning = () => (
  <Input placeholder="Hello World!" borderColor="warning" />
)

export const disabled = () => <Input disabled placeholder="Hello World!" />
