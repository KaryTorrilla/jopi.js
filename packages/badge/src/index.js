import React from 'react'
import { Box } from '@oneloop/box'
import '@oneloop/fonts'

export const Badge = ({ isMedium, hasIcon, isNotButton, ...props }) => (
  <Box
    as={ isNotButton ? "" : "button" }
    tx="buttons"
    variant="primary"
    {...props}
    __css={{
      lineHeight: '14px',
      height: isMedium ? '22px' : '16px',
      display: 'inline-block',
      fontSize: '10px',
      fontFamily: 'primary',
      textAlign: 'center',
      textTransform: 'uppercase',
      fontWeight: 'bold',
      letterSpacing: '0.05em',
      px: 2,
      py: '2px',
      color: 'white',
      bg: 'primary',
      border: 0,
      borderRadius: isMedium ? '14px' : '8px',
      flexDirection: 'row',
      justifyContent: 'center',
      alignItems: 'center',
      padding: hasIcon ? '4px 8px' : '1px 5px',
      position: 'static',
      flex: 'none',
      order: 0,
      flexGrow: 0,
      margin: '0px 0px',
    }}
  />
)
