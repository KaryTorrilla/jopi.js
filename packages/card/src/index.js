import React from 'react'
import { Box } from '@oneloop/box'
import theme from '@oneloop/theme'

export const Card = ({ variant, active, ...props }) => {
  const variantValues = Object.values(theme.card)[Object.keys(theme.card).indexOf(variant)]
  const backgroundColor = active ? variantValues[':focus'].backgroundColor : variantValues.backgroundColorValue
  return (
    <Box sx={{ position: 'relative' }}>
      <Box
        tx='card'
        variant={variant}
        {...props}
        __css={{
          fontFamily: 'primary',
          display: 'flex',
          flexDirection: 'row',
          alignItems: 'center',
          backgroundColor: backgroundColor,
          position: 'static',
        }}
        />
    </Box>
  )
}
