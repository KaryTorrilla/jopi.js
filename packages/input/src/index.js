import React from 'react'
import PropTypes from 'prop-types'
import { Box } from '@oneloop/box'
import { Text } from '@oneloop/text'
import { useToggle } from '@oneloop/hooks'
import theme from '@oneloop/theme'

export const Input = ({ prefix, suffix, label, errors, variant = "input", variantSize = "inputLarge", infoAlert, disabled, readonly, inline, password, ...props }) => {
  const [hover, setHover] = useToggle(false)
  const [focused, setFocused] = useToggle(false)
  const [text, setText] = React.useState('')
  const indexVariant = Object.keys(theme.forms).indexOf(variant)
  
  let finalVariant = variant+".normal"
  let colorPlaceholder = Object.values(theme.forms)[indexVariant].normal.color
  if (hover) {
    finalVariant = variant+".hover"
    colorPlaceholder = Object.values(theme.forms)[indexVariant].hover.color
  }
  if (infoAlert != undefined){
    finalVariant = "inputFocus"
  } else if (errors != undefined) {
    finalVariant = "inputError"
    colorPlaceholder = "error"
  }
  
  if (focused) {
    finalVariant = variant+".active"
    colorPlaceholder = Object.values(theme.forms)[indexVariant].active.color
  } else if (!focused && text != ''){
    finalVariant = variant+".filled"
    colorPlaceholder = Object.values(theme.forms)[indexVariant].filled.color
  } else if (disabled) {
    finalVariant = "inputDisabled"
    colorPlaceholder = "neutralGray5"
  } else if (readonly) {
    finalVariant = "inputReadonly"
    colorPlaceholder = "neutralGray1"
  }

  const index = Object.keys(theme.forms).indexOf(variantSize)
  const paddingLeft = prefix ? Object.values(theme.forms)[index].paddingLeftPrefix : Object.values(theme.forms)[index].paddingLeftNoPrefix
  const fontSizePrefix = Object.values(theme.forms)[index].fontSize

  return (
    <Box
      __css={{
        display: inline && 'flex',
        flexDirection: inline && 'row',
        alignItems: inline && 'center',
        gap: inline && '16px',
      }}
    >
      {label && (
        <Box
          __css={{
            mb: '4px'
          }}
        >
          <Text variant="body600.fontSize12">
            {label}
          </Text>
        </Box>
      )}
      <Box
        __css={{
          position: 'relative',
          height: 'fit-content',
          display: 'flex',
          flexDirection: 'row',
          alignItems: 'center',
          gap: '6px',
          color: colorPlaceholder,
          '> *:first-child': prefix
            ? { position: 'absolute', height: fontSizePrefix, fontSize: fontSizePrefix, left: '7px', mt: '2px' }
            : {},
          '> *:last-child': suffix
            ? { position: 'absolute', top: '9px', right: '12px' }
            : {},
        }}
      >
        {prefix}
        <Box
          as="input"
          type={ password ? "password" : "text" }
          tx="forms"
          variant={[finalVariant, variantSize]}
          onMouseOver={() => setHover(true)}
          onMouseOut={() => setHover(false)}
          onChange={e => setText(e.target.value)}
          onFocus={() => setFocused(true)}
          onBlur={() => setFocused(false)}
          {...props}
          __css={{
            border: 'none',
            boxShadow: 'none',
            fontFamily: 'primary',
            display: 'block',
            paddingLeft: paddingLeft,
            outline: 'none',
          }}
        />
        {suffix}
      </Box>
      {errors && (
        <Box
          __css={{
            px: 16,
            py: 1,
          }}
        >
          <Text variant="inputMessageErrors">{errors}</Text>
        </Box>
      )}
      {infoAlert && (
        <Box
          __css={{
            px: 16,
            py: 1,
          }}
        >
          <Text variant="inputMessageInfoAlert">{infoAlert}</Text>
        </Box>
      )}
    </Box>
  )
}

Input.propTypes = {
  placeholder: PropTypes.string,
}

Input.defaultProps = {
  disabled: false,
}
