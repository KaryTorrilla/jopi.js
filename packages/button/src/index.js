import React from 'react'
import { Badge } from '@oneloop/badge'
import { Box } from '@oneloop/box'
import { Checkbox } from '@oneloop/checkbox'
import { Icon } from '@oneloop/icons'
import { Image } from '@oneloop/image'
import { useToggle } from '@oneloop/hooks'
import theme from '@oneloop/theme'
import '@oneloop/fonts'

export const Button = ({ variant, ...props }) => (
  <Box sx={{ position: 'relative' }}>
    <Box
      as='button'
      tx='buttons'
      variant={variant}
      {...props}
      __css={{
        appearance: 'none',
        display: 'flex',
        lineHeight: 'inherit',
        fontFamily: 'Nunito Sans',
        fontWeight: 'normal',
        textAlign: 'center',
        textDecoration: 'none',
        cursor: 'pointer',
        outline: 'none',
        color: 'white',
        bg: 'primary',
        border: 0,
        borderRadius: '12px',
        flexDirection: 'row',
        justifyContent: 'center',
        alignItems: 'center',
        whiteSpace: 'nowrap',
      }}
    />
  </Box>
)

export const ButtonIcon = ({ icon, variant, text, badgeValue = 0, badgeVariant = 'primary', maxWidth, ...props }) => {
  let fontSize
  if (Array.isArray(variant)) {
    const indexes = variant.map(v => Object.keys(theme.buttons).indexOf(v))
    indexes.map(index => {
      if (Object.values(theme.buttons)[index].fontSize !== undefined) {
        fontSize = Object.values(theme.buttons)[index].fontSize
      }
      return fontSize
    })
  } else {
    const index = Object.keys(theme.buttons).indexOf(variant)
    fontSize = Object.values(theme.buttons)[index].fontSize
  }

  return (
    <Box sx={ maxWidth ? { position: 'relative', width: '100%' } : { position: 'relative' }}>
      <Box
        as='button'
        tx='buttons'
        variant={variant}
        {...props}
        __css={{
          appearance: 'none',
          display: 'flex',
          lineHeight: 'inherit',
          fontFamily: 'Nunito Sans',
          fontWeight: 'bold',
          textAlign: 'center',
          textDecoration: 'none',
          cursor: 'pointer',
          outline: 'none',
          border: 0,
          flexDirection: 'row',
          justifyContent: 'center',
          alignItems: 'center',
          whiteSpace: 'nowrap',
          position: 'relative',
        }}
      >
        { icon && <Icon icon={icon} fontSize={fontSize}/> }
        { badgeValue !== 0 && <Badge variant={badgeVariant} isNotButton style={{ position: 'absolute', top: '2px', left: '16px' }}>{ badgeValue }</Badge> }
        { text && <span>{text}</span> }
      </Box>
    </Box>
  )
}

export const ButtonHoldPress = ({ variant, active = false, text, icon, badgeValue = 0, badgeVariant = 'primary', isCollapsible, hasCheckbox = false, disabled = false, maxWidth, ...props }) => {
  let colorValue
  let colorValueIcon
  let fontSize
  let fontWeight
  if (Array.isArray(variant)) {
    const indexes = variant.map(v => Object.keys(theme.buttons).indexOf(v))
    indexes.map(index => {
      if (Object.values(theme.buttons)[index][':focus'] !== undefined && active) {
        colorValue = Object.values(theme.buttons)[index][':focus'].color
        fontWeight = Object.values(theme.buttons)[index][':focus'].fontWeight
        if (Object.values(theme.buttons)[index].colorIcon !== undefined) {
          colorValueIcon = Object.values(theme.buttons)[index][':focus'].colorIcon
        }
      } else if (Object.values(theme.buttons)[index].color !== undefined) {
        colorValue = Object.values(theme.buttons)[index].color
      } else if (Object.values(theme.buttons)[index].colorIcon !== undefined) {
        colorValueIcon = Object.values(theme.buttons)[index].colorIcon
      } else if (Object.values(theme.buttons)[index].fontWeight !== undefined) {
        fontWeight = Object.values(theme.buttons)[index].fontWeight
      }
      if (Object.values(theme.buttons)[index].fontSize !== undefined) {
        fontSize = Object.values(theme.buttons)[index].fontSize
      }
      return colorValue
    })
  } else {
    const index = Object.keys(theme.buttons).indexOf(variant)
    colorValue = Object.values(theme.buttons)[index].color
    colorValueIcon = Object.values(theme.buttons)[index].colorIcon
    fontSize = Object.values(theme.buttons)[index].fontSize
    fontWeight = Object.values(theme.buttons)[index].fontWeight
    if (active) {
      colorValue = Object.values(theme.buttons)[index][':focus'].color
      colorValueIcon = Object.values(theme.buttons)[index][':focus'].colorIcon
      fontWeight = Object.values(theme.buttons)[index][':focus'].fontWeight
    }
  }

  return (
    <Box sx={ maxWidth ? { position: 'relative', width: '100%' } : { position: 'relative' }}>
      <Box
        as='button'
        tx='buttons'
        variant={variant}
        {...props}
        __css={{
          appearance: 'none',
          display: 'flex',
          lineHeight: 'inherit',
          fontFamily: 'Nunito Sans',
          fontWeight: {fontWeight},
          textAlign: 'center',
          textDecoration: 'none',
          cursor: 'pointer',
          outline: 'none',
          border: 0,
          flexDirection: 'row',
          //  justifyContent: 'center',
          width: maxWidth &&'100%',
          alignItems: 'center',
          whiteSpace: 'nowrap',
        }}
      >
        { hasCheckbox && active && <Checkbox defaultChecked disabled={disabled}/>}
        { hasCheckbox && !active && <Checkbox defaultChecked={false} disabled={disabled}/>}
        { icon && <Icon icon={icon} fontSize={fontSize} style={{ color: ( colorValueIcon || colorValue ) }}/> }
        { text && <span style={colorValue && { color: colorValue, fontWeight: fontWeight }}>{text}</span>}
        { badgeValue === 0 && isCollapsible && <span className='icon-dropdown' style={{ position: 'absolute', right: '10px', color: '#798B97', fontSize: '14px', transform: active ? 'rotate(-180deg)' : 'rotate(0deg)', paddingTop: '4px' }}/> }
        { badgeValue !== 0 && !text && <Badge variant={badgeVariant} isNotButton style={{ position: 'absolute', top: '2px', left: '16px' }}>{badgeValue}</Badge> }
        { badgeValue !== 0 && text && <Badge variant={badgeVariant} isNotButton style={{ position: 'absolute', right: '16px' }}>{badgeValue}</Badge> }
      </Box>
    </Box>
  )
}

export const ButtonUser = ({ variant, srcImage, text, ...props }) => (
  <Box sx={{ position: 'relative' }}>
    <Box
      as='button'
      tx='buttons'
      variant={variant}
      {...props}
      __css={{
        appearance: 'none',
        display: 'flex',
        lineHeight: 'inherit',
        fontFamily: 'Nunito Sans',
        fontWeight: 'bold',
        fontSize: '14px',
        textAlign: 'center',
        textDecoration: 'none',
        cursor: 'pointer',
        outline: 'none',
        color: 'white',
        bg: 'primary',
        border: 0,
        borderRadius: '12px',
        flexDirection: 'row',
        justifyContent: 'center',
        alignItems: 'center',
        height: '48px',
        whiteSpace: 'nowrap',
        padding: '13px 26px 13px 26px',
        gap: '8px',
      }}
    >
      {srcImage && <Image src={srcImage} variant="avatar"/> }
      {!srcImage && <span className='icon-contacto' style={{ fontSize: '24px', height: '24px' }}/>}
      {text}
    </Box>
  </Box>
)
