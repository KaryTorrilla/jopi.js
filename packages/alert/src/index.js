import React from 'react'
import { Flex } from '@oneloop/box'
import { Check, Exclamation, Times } from '@oneloop/icons'
import { Text } from '@oneloop/text'
import { Button } from '@oneloop/button'

export const Alert = ({ type = 'success', text, ...props }) => (
  <Flex
    variant="alert"
    {...props}
    __css={{
      width: '100%',
      bg: 'white',
      borderRadius: 'circle',
      boxShadow: 'active',
      justifyContent: 'space-between',
      p: 1,
    }}
  >
    {type === 'success' && (
      <Check
        bg="success"
        color="white"
        borderRadius="circle"
        size={40}
        p={3}
        mr={5}
      />
    )}
    {type === 'warning' && (
      <Exclamation
        bg="warning"
        color="white"
        borderRadius="circle"
        size={40}
        p={3}
        mr={5}
      />
    )}
    {type === 'danger' && (
      <Times
        bg="danger"
        color="white"
        borderRadius="circle"
        size={40}
        p={3}
        mr={5}
      />
    )}
    <Text variant="body.0" display="flex" alignItems="center" flexGrow={1}>
      {text}
    </Text>
    <Button
      size={32}
      bg="#F6F2F2"
      p={3}
      display="inline-flex"
      alignSelf="center"
      mr={2}
    >
      <Times color="#716F6F" m="auto" />
    </Button>
  </Flex>
)
