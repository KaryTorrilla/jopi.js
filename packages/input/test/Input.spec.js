import React from 'react'
import renderer from 'react-test-renderer'
import { Icon } from '@oneloop/icons'
import { Box } from '@oneloop/box'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import 'jest-styled-components'

import { Input } from '../src'

Enzyme.configure({ adapter: new Adapter() })

describe('Input', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Input />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('placeholder', () => {
    const tree = renderer.create(<Input placeholder="test" />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('disabled', () => {
    const tree = renderer.create(<Input disabled />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('readonly', () => {
    const tree = renderer
      .create(<Input readonly placeholder="Hello World!" />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('errors', () => {
    const tree = renderer
      .create(
        <Input errors="Minimo 8 caracteres." placeholder="Placeholder text" />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('infoAlert', () => {
    const tree = renderer
      .create(
        <Input infoAlert="Campo Requerido" placeholder="Placeholder text" />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('inputMedium', () => {
    const tree = renderer
      .create(
        <Input placeholder="Placeholder text" variantSize="inputMedium" />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('inputWithIcon', () => {
    const tree = renderer
      .create(
        <Input
          prefix={<span className="icon-email" />}
          placeholder="Hello World!"
        />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('inputWithLabel', () => {
    const tree = renderer
      .create(<Input label="Label de prueba" placeholder="Hello World!" />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('inputInlineLabel', () => {
    const tree = renderer
      .create(<Input label="Label" placeholder="Hello World!" inline />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('inputWithPassword', () => {
    const tree = renderer
      .create(
        <Input
          suffix={<span className="icon-ocultar" />}
          label="Label de prueba"
          placeholder="Hello World!"
          password
        />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('inputWithPasswordIcon', () => {
    const tree = renderer
      .create(
        <Input
          suffix={<Icon icon="icon-ocultar" />}
          label="Label de prueba"
          placeholder="Hello World!"
          password
        />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('searchBold', () => {
    const tree = renderer
      .create(
        <Input
          prefix={<span className="icon-buscar" />}
          placeholder="Search"
          variant="inputSearch"
          bold
        />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('searchBoldIcon', () => {
    const tree = renderer
      .create(
        <Input
          prefix={<Icon icon="icon-buscar" />}
          placeholder="Search"
          variant="inputSearch"
          bold
        />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('checkStates', () => {
    const component = shallow(<Input placeholder="Placeholder text" />)
    expect(component.find(<Box as="input" />).exists).toBeTruthy()

    const newInputValue = 'test'
    component
      .find(Box)
      .at(2)
      .simulate('change', { target: { value: newInputValue } })
    component.find(Box).at(2).simulate('mouseover')
    component.find(Box).at(2).simulate('mouseout')
    component.find(Box).at(2).simulate('focus')
    component.find(Box).at(2).simulate('blur')

    expect(component).toMatchSnapshot()
  })
})
