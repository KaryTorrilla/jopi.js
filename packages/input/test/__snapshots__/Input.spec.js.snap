// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input disabled 1`] = `
.c0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
}

.c1 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  border-radius: 4px;
  border: 1px solid #DADADA;
  box-shadow: none;
  caret-color: #757575;
  color: #757575;
  display: block;
  font-size: 14px;
  height: 36px;
  line-height: 14;
  padding-left: 16px;
  padding-right: 16px;
  padding-top: 11px;
  padding-bottom: 11px;
  margin-bottom: 64px;
  outline: none;
}

.c1:disabled {
  color: #999999;
  background-color: #EBEBEB;
  cursor: not-allowed;
}

<div
  className="c0"
>
  <input
    className="c1"
    disabled={true}
    type="text"
  />
</div>
`;

exports[`Input placeholder 1`] = `
.c0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
}

.c1 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  border-radius: 4px;
  border: 1px solid #DADADA;
  box-shadow: none;
  caret-color: #757575;
  color: #757575;
  display: block;
  font-size: 14px;
  height: 36px;
  line-height: 14;
  padding-left: 16px;
  padding-right: 16px;
  padding-top: 11px;
  padding-bottom: 11px;
  margin-bottom: 64px;
  outline: none;
}

.c1:disabled {
  color: #999999;
  background-color: #EBEBEB;
  cursor: not-allowed;
}

<div
  className="c0"
>
  <input
    className="c1"
    disabled={false}
    placeholder="test"
    type="text"
  />
</div>
`;

exports[`Input renders correctly 1`] = `
.c0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
}

.c1 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  border-radius: 4px;
  border: 1px solid #DADADA;
  box-shadow: none;
  caret-color: #757575;
  color: #757575;
  display: block;
  font-size: 14px;
  height: 36px;
  line-height: 14;
  padding-left: 16px;
  padding-right: 16px;
  padding-top: 11px;
  padding-bottom: 11px;
  margin-bottom: 64px;
  outline: none;
}

.c1:disabled {
  color: #999999;
  background-color: #EBEBEB;
  cursor: not-allowed;
}

<div
  className="c0"
>
  <input
    className="c1"
    disabled={false}
    type="text"
  />
</div>
`;
